CallsheetLocalizations/pl.xcloc/Localized Contents/pl.xliff

```diff
@@ -677,10 +709,12 @@
       </trans-unit>
       <trans-unit id="I'm in a popover" xml:space="preserve">
         <source>I'm in a popover</source>
+        <target>I'm in a popover</target>
         <note>Do not translate</note>
       </trans-unit>
       <trans-unit id="IAP: Monthly Description" xml:space="preserve">
         <source>Unlimited searches for a month. Auto-renews</source>
+        <target>Nieograniczone wyszukiwania przez miesiąc. Automatycznie odnawia subskrypcję</target>
```

Comment:
Może "automatycznie odnawiane"? trochę długie się to zrobiło, może się nie zmieścić (zwłaszcza u tych, którzy jak ja powiększają fonty)